src/shapes/MoviesListShapes.js

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+export const MoviesListShape = {
+  movies: PropTypes.arrayOf(
+    PropTypes.shape({
+      imdbId: PropTypes.string.isRequired,
+    }),
+  ).isRequired,
```

Comment:
import MovieCardShape here and use in MoviesListShape

```suggestion
export const MoviesListShape = {
  movies: PropTypes.arrayOf(MovieCardShape),
  ).isRequired,
```